packages/@lwc/engine-server/README.md

```diff
@@ -13,14 +13,20 @@ This package supports the following APIs.
 
 This function renders a string-representation of a serialized component tree, given a tag name
 and an LWC constructor. The output format itself is aligned with the [current leading
-proposal][explainer], but is subject to change.
+proposal][explainer], but is subject to change. The result is a pair {html,styles} with
+html being the markup, whyle styles is undefined.
+
```

Comment:
Let's keep the code sample here for native shadow and create a new one for synthetic shadow.

```suggestion
import { renderComponent } from 'lwc';
import LightningHello from 'lightning/hello';

const componentProps = {};
const { html, styles } = renderComponent('lightning-hello', LightningHello, componentProps);
```